### DISCUSSION TITLE
Invalid options object. Image Minimizer Plugin has been initialized using an options object that does not match the API schema.

### DISCUSSION THREAD

**Discussion by Niyabi** [webpack/webpack#14156] [Q&A] [answered]
I wanted to configure webpack so it can minimize images so I decided to use [ImageMinimizerWebpackPlugin](https://webpack.js.org/plugins/image-minimizer-webpack-plugin/). I chose to optimize with squoosh and I did it like in documentation but when I try to build project I get error:

`Invalid options object. Image Minimizer Plugin has been initialized using an options object that does not match the API schema. options has an unknown property 'minify'. These properties are valid: object { test?, include?, exclude?, minimizerOptions?, filter?, severityError?, loader?, maxConcurrency?, filename?, deleteOriginalAssets? }`

So as Image Minimizer Plugin does not have such option then how should I configure it? I guess I should use loader but how to combine it with squoosh?

**Comment by alexander-akait** [comment, marked as answer]
Because it is not released yet

**Reply by alexander-akait** [reply]
`squoosh` still has the problems, we will do release, when it was fixed
